clmm/core/manager.py

```diff
@@ -0,0 +1,112 @@
+'''
+Manager makes the interactions between GalaxyCluster and the other clmm objects
+The Manager class talks to galaxy_cluster.GalaxyCluster objects (which can have several types, observed, simulated, different datasets, etc.) and talks with other functions/methods etc that will use the GalaxyCluster objects, e.g. inferring the mass, measuring profiles.
+
+'''
+from clmm.core.datatypes import GCData, find_in_datalist
+
+class Manager():
+
+    '''
+    Makes the interactions between GalaxyCluster and the other clmm objects
+    '''
+
+    def __init__(self):
+        '''
+        '''
+        pass
```

Comment:
Should this `pass` be here? There is code after it

Reply:
removed